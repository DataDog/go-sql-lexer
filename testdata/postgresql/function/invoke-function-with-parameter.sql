SELECT get_user_email(1);
