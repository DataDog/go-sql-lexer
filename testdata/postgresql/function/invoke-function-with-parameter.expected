[
    {
      "expected": "SELECT get_user_email ( ? )"
    }
  ]
  